.travis.yml

```diff
@@ -15,4 +15,4 @@ install:
 script:
   - vendor/bin/phpunit
   - bin/psalm
-  - vendor/bin/phpcs
+  - vendor/bin/php-cs-fixer fix --verbose --allow-risky=yes --dry-run
```

Comment:
@SignpostMarv 
it's cool that config file is choosing same file than other SCA tools.

My suggestion here is that in runtime, at travis, you don't need to investigate all files.
you could keep setting file as is, and on travis runtime, ask to validate only modified files.